sw/host/opentitanlib/src/test_utils/bitbanging/mod.rs

```diff
@@ -0,0 +1,5 @@
+// Copyright lowRISC contributors.
```

Comment:
FYI: `// Copyright lowRISC contributors (OpenTitan project).`

Reply:
Nevermind, this isn't on `earlgrey_es_sival`. If you cherry pick to master, you'll need to change it.